javers-core/src/main/java/org/javers/core/Javers.java

```diff
@@ -54,10 +55,12 @@
      */
     Commit commit(String author, Object currentVersion);
 
+    Commit commit(String author, Map<String, String> properties, Object currentVersion);
```

Comment:
imo ładniej będzie jeśli properties jako optional parameter będą na końcu
```
Commit commit(String author, Object currentVersion);
Commit commit(String author, Object currentVersion, Map<String, String> properties);
```
no i może krótki javadoc to nowego parametru properties?